go.mod

```diff
@@ -224,7 +224,7 @@ require (
 	github.com/stretchr/objx v0.3.0 // indirect
 	github.com/subosito/gotenv v1.2.0 // indirect
 	github.com/sylvia7788/contextcheck v1.0.4 // indirect
-	github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca // indirect
+	github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca
```

Comment:
What's the reason behind changing goleveldb from inderect -> direct import?

Reply:
I think this change not from my PR. `go.mod` have same change in main branch when i run `go mod tidy`.